src/main/java/com/bookstore/bookstoreapi/data/entity/Customer.java

```diff
@@ -41,6 +41,9 @@ public class Customer {
   @Column(name = "Address", nullable = false)
   private String address;
 
+  @Column(name = "Ratings", nullable = true) //nullable = true since the user doesn't HAVE to leave ratings?
+  private ArrayList<Rating> ratings = new ArrayList<>();
```

Comment:
In this case you just need a relationship between the Rating entity and Customer entity.
Also since you are retrieving BookID, you will need a relationship with Rating and Book entity.
Please look at the comments below.